burnman/solidsolution.py

```diff
@@ -33,7 +33,14 @@ class SolidSolution(Mineral):
     and P derivatives in J/K/mol and m^3/mol.
     """
 
-    def __init__(self, molar_fractions=None):
+    def __init__(self,
```

Comment:
one option to make it compatible if you make molar_fractions the first argument. Thoughts?